easybuild/easyconfigs/v/vaspkit/vaspkit-1.3.3.eb

```diff
@@ -0,0 +1,33 @@
+# IT4INNOVATIONS 2022
+# JK
+
+easyblock = 'Tarball'
+
+name = 'vaspkit'
+version = '1.3.3'
+
+homepage = 'https://vaspkit.com/'
+description = """VASPKIT aims at providing a powerful and user-friendly interface to perform high
+ throughput analysis of various material properties from the raw calculated data using the widely-used
+ VASP code."""
+
+toolchain = SYSTEM
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(namelower)s.%(version)s.linux.x64.tar.gz']
+patches = ['vaspkit-1.3.3-fix-setup-script.patch']
+checksums = [
+    'e428d5e3f3dde5f11fc17770f7c0864a',  # vaspkit.1.3.3.linux.x64.tar.gz
+    'f4f523c04d27c9f2165b140a0c0ccde0',  # vaspkit-1.3.3-fix-setup-script.patch
+]
+extract_sources = True
+
+dependencies = [
+    ('matplotlib', '3.3.3', '', ('foss', '2020b')),
+]
+
+skipsteps = ['sanitycheck']
+
+modluafooter = 'execute{cmd="bash $EBROOTVASPKIT/setup.sh", modeA={"load"}}'
```

Comment:
Hmm, are you sure this actually works? What does the `setup.sh` script do? Does it change the environment?
Because I don't think that's going to work at all...

Also, what if EasyBuild is configured to generate module files in Tcl syntax instead?

Reply:
> Hmm, are you sure this actually works? What does the `setup.sh` script do? Does it change the environment? Because I don't think that's going to work at all...

This is the vaspkit way of doing stuff, though it might be probably better to export most of the script into the module file itself. However, there's also .vaspkit file created in the /home/$USER directory, which includes the rest of the configuration (which the user will have to change around according to their needs). I'll try to write a patch that would circumvent most of this script.

> Also, what if EasyBuild is configured to generate module files in Tcl syntax instead?

I have no experience with the Tcl, so I cannot attest to this.